cmd/admin.go

```diff
@@ -57,8 +58,59 @@ to make automatic initialization process more smoothly`,
 			},
 		},
 	}
+
+	subcmdChangePassword = cli.Command{
+		Name:   "change-password",
```

Comment:
No reason, this was suggested by @bkcsoft on IRC if I recall correctly, and I thought it was a clean and self-describing name